.github/workflows/pytest.yaml

```diff
@@ -98,7 +98,7 @@ jobs:
 
       # test deploy ----
       - name: "Test Deploy to PyPI"
-        uses: pypa/gh-action-pypi-publish@release/v1
+        uses: pypa/gh-action-pypi-publish@v1
```

Comment:
Are you sure this is right? I see a branch named `release/v1`, but not one named `v1`. (And same for the other instance below.)

Reply:
```suggestion
        uses: pypa/gh-action-pypi-publish@release/v1
```